pymeasure/instruments/mock.py

```diff
@@ -100,3 +106,52 @@ def set_output_voltage(self, value):
         self._output_voltage = value
 
     output_voltage = property(fget=get_output_voltage, fset=set_output_voltage)
+
+    def get_frame_width(self):
```

Comment:
Is there a specific reason you're not using the `@property` decorator that [comes with Python](https://docs.python.org/3.9/library/functions.html?highlight=@property#property)? That would make for a more concise definition of these methods, and would not add e.g. the `set_frame_width` to the class namespace.

Reply:
I was just following the format that properties were already written in the class before. I agree that using the `@property` decorator is cleaner though. I've moved all of the properties to use this.